datahub-web-react/src/app/policy/PoliciesPage.tsx

```diff
@@ -239,10 +453,8 @@ export const PoliciesPage = () => {
                 <PolicyDetailsModal
                     policy={focusPolicy}
                     visible={showViewPolicyModal}
-                    onEdit={onEditPolicy}
                     onClose={onCancelViewPolicy}
-                    onRemove={onRemovePolicy}
-                    onToggleActive={onToggleActive}
+                    privileges={privileges}
```

Comment:
what if we had a prop called 

"getPrivilegeDisplayName(type: string) => string"